spring-cloud-aws-dynamodb/src/main/java/io/awspring/cloud/dynamodb/DynamoDbTableSchemaResolver.java

```diff
@@ -35,4 +35,13 @@ public interface DynamoDbTableSchemaResolver {
 	 * @param <T> - type
 	 */
 	<T> TableSchema resolve(Class<T> clazz, String tableName);
+
+	/**
+	 * Register manually a {@link TableSchema}
+	 *
+	 * @param tableSchema - the schema to be registered
+	 * @param tableName - the table name
+	 */
+	<T> void register(TableSchema<T> tableSchema, String tableName);
```

Comment:
We should add default implementation so we are not making breaking changes.

Reply:
done

Reply:
We are in RC so breaking changes are fine. Having this as default empty is quite unexpected from the user side..?

Reply:
Well it depends I would say. Our `DefaultDynamoDbTableNameResolver` is using caching with a map which has concrete implementation. If someone wants to implement their own `DynamoDbTableNameResolver`, they shouldn't be forced to cache. I mean from design perspective it should be used for caching imho. If we can do breaking change I am fine with both solution.

Reply:
@maciejwalkowiak  @MatejNedic  should I rollback the last commit or we'll go as it is?